spring-cloud-skipper-docs/src/main/asciidoc/api-guide.adoc

```diff
@@ -137,6 +137,286 @@ include::{snippets}/about-documentation/get-meta-information/http-response.adoc[
 
 include::{snippets}/about-documentation/get-meta-information/response-fields.adoc[]
 
+[[resources-upload]]
+=== Upload
+
+==== Upload a package
+
+TBD
+
+===== Request structure
+
+include::{snippets}/upload-documentation/upload-release/http-request.adoc[]
+
+===== Example request
+
+include::{snippets}/upload-documentation/upload-release/curl-request.adoc[]
+
+===== Response structure
+
+include::{snippets}/upload-documentation/upload-release/http-response.adoc[]
+
+===== Response fields
+
+include::{snippets}/upload-documentation/upload-release/response-fields.adoc[]
+
+[[resources-install]]
+=== Install
+
+==== Install a package
+
+TBD
+
+===== Request structure
+
+include::{snippets}/install-documentation/install-package/http-request.adoc[]
+
+===== Example request
+
+include::{snippets}/install-documentation/install-package/curl-request.adoc[]
+
+===== Response structure
+
+include::{snippets}/install-documentation/install-package/http-response.adoc[]
+
+===== Response fields
+
+include::{snippets}/install-documentation/install-package/response-fields.adoc[]
+
+==== Install a package with id
+
+TBD
+
+===== Request structure
+
+include::{snippets}/install-documentation/install-package-with-id/http-request.adoc[]
```

Comment:
I guess this may be the answer to my question here: https://github.com/spring-cloud/spring-cloud-skipper/pull/172#discussion_r144724737. Now that REST endpoints use `/install/id` as the URI identifier and for few other REST endpoints, should these be changed as well?

Reply:
After generating doc locally, I could confirm the naming `install-package-with-id` doesn't matter. The generated doc looks great :-). I will try to fix the TBD places and merge this PR.